spec/controllers/state_file/archived_intake/archived_intake_controller_spec.rb

```diff
@@ -3,7 +3,8 @@
 describe StateFile::ArchivedIntakes::ArchivedIntakeController, type: :controller do
   let(:ip_address) { '192.168.0.1' }
   let(:email_address) { 'test@example.com' }
-  let!(:request_instance) { create :state_file_archived_intake_request, ip_address: ip_address, email_address: email_address }
+  let(:intake) {create :state_file_archived_intake}
```

Comment:
[nit] can we call this an `archived_intake` since we use `intake` to refer to a completely different db model.